types/ioredis/index.d.ts

```diff
@@ -323,8 +323,8 @@ declare namespace IORedis {
         hvals(key: KeyType, callback: (err: Error, res: string[]) => void): void;
         hvals(key: KeyType): Promise<string[]>;
 
```

Comment:
That should be `{ [key: string]: string | number }` iirc

Reply:
fixed it

Reply:
This is completely wrong. Should I create another pull request fixing an already merged pull request?

Reply:
No you would open a new one.
My bad for suggesting the wrong type here, got it mixed up with something else.